### DISCUSSION TITLE
Panorama: Allow for setting specific post width

### DISCUSSION THREAD

**Discussion by magesnmiceys** [AprilSylph/XKit-Rewritten#853] [Suggestion Box]
Someone's suggested this before back in January, but I haven't really seen anything come out of it.

Allowing for a maximum post width would be super nice.  Only problem with Panorama is that images get stretched out really far, so they _also_ become really tall and it gets a bit silly to try and see them this way.

Suggested solution to this is either a maximum width for posts or an option for specific image sizes.

**Comment by marcustyphoon** [comment]
Working on it! #845
